packages/core-web/src/index.js

```diff
@@ -161,9 +165,12 @@ function setup() {
   // eslint-disable-next-line no-unused-vars
   const vm = new Vue({
     el: '#app',
-    created() {
-      executeAfterCreatedRoutines();
+    render(createElement) {
+      // eslint-disable-next-line no-undef
```

Comment:
how about `/* global ... */`? https://eslint.org/docs/rules/no-undef

Reply:
Thanks for the tip! I've updated it accordingly :-)